### PR TITLE
PolyhedralGeometry: Deal with fine and star triangulations.

### PR DESCRIPTION
**Pull request by lkastner**
The necessary functionality in polymake cannot be accessed directly right now due to an incomplete bugfix. The ugly filter methods will go away after polymake 4.7 release.

### CODE REVIEW

File: src/PolyhedralGeometry/triangulations.jl [hunk 1]
```diff
@@ -41,7 +72,7 @@ end
 
 Compute all triangulations that can be formed using the vertices of the given
 polytope `P`.
-
+p
```
Comment (fingolfin) [inline src/PolyhedralGeometry/triangulations.jl]:
```suggestion

```

File: src/PolyhedralGeometry/triangulations.jl [hunk 2]
```diff
@@ -59,18 +90,82 @@ julia> all_triangulations(c)
...
+"""    
+function star_triangulations(pts::AnyVecOrMat; full::Bool=false, regular::Bool=false)
+    if regular
+        result = regular_triangulations(pts; full=full)
+    else
+        result = all_triangulations(pts; full=full)
+    end
+    return [t for t in result if _is_star_triangulation(t)]
+end
+@doc Markdown.doc"""
```
Comment (fingolfin) [inline src/PolyhedralGeometry/triangulations.jl:120]:
```suggestion

@doc Markdown.doc"""
```

**Review by fingolfin — COMMENTED** [review]
Which ones are the "ugly filter methods will go away after polymake 4.7 release"? Perhaps add a comment to each of them stating just that?

**Comment by lkastner** [comment]
I don't understand this failure.

**Comment by lkastner** [comment]
> I don't understand this failure.

And now it is gone?

**Review by fingolfin — APPROVED** [review]

**Comment by HereAround** [comment]
Sorry for the late response. This looks great. Thank you @lkastner!

**Pull request merged**